src/core/lru.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+#pragma once
+
+#include <absl/container/flat_hash_map.h>
+
+namespace dfly {
+
+using namespace std;
+
+struct DLinkedNode {
```

Comment:
I am still not bought why do we need LRUCache (even if it's so simple) when we have dash_table.
Especially if DashTable can easily become a cache - see https://www.romange.com/2022/06/23/dragonfly-cache-design/

I can approve this code as temporary but then I will probably replace it with DashTable - which won't not require allocations during insert path.

Reply:
And even if we say - Dashtable is over-engineering for this use-case - fine, but I would prefer a preallocated array of capacity elements than a dynamic linked list.